notebooks/WFC3/calwf3_v1.0_cte/pre-requirements.sh

```diff
@@ -1 +1 @@
-conda install --yes -c http://ssb.stsci.edu/astroconda hstcal==2.5.0
```

Comment:
> An older version of HSTCAL is required for the notebook so this doesn't need to be updated.

In such cases, should this notebook be rewritten or perhaps deprecated? What do you think about this, @sosey?

Reply:
Apologies, I should have been more specific. The purpose of the notebook is to showcase a particular advantage `hstcal==2.5.0` has over recent versions. So it's not that the notebook can't run without the recent version (it will), it's that the notebook needs the older version to showcase that advantage. Does that help clarify things?